### ISSUE TITLE
[RFC] Order methods in class by visibility

### ISSUE THREAD

**Issue by umpirsky** [FriendsOfPHP/PHP-CS-Fixer#1294]
Would be nice to have a fixer to order methods in class by visibility: public, protected, private.

**Comment by keradus** [comment]
Ref #832

**Comment by funivan** [comment]
how to deal with this structure?
```php
public function test1(){}
private function test2(){}
const A = 1;
public function test3(){}
```
Move `const` to the end or move all `methods` to the end.

Possible it will be better to create single fixer witch order constant properties and etc
See phpstorm rearranger http://blog.jetbrains.com/webide/2013/03/arrange-your-code-automatically-with-phpstorm-rearranger/

**Comment by keradus** [comment]
This fixer must be run after #832

**Comment by keradus** [comment]
Or not relay on it but have priority to run after it, and if one not chosen to run fixer from #832 then just reorder consecutive methods, so reorder test1 & test2 if needed, then reorder test3 (and test4 if exists)

**Comment by keradus** [comment]
Closing due to merge of #1628
